loadtest-list.py

```diff
@@ -16,8 +16,8 @@
 NUM_SAMPLE_SHOTS = random.randint(4, 7)
 NUM_SEARCH_HITS = 2
 
-WEIGHT_LIST_SHOTS = int(os.getenv('WEIGHT_LIST_SHOTS', 0))
-WEIGHT_SEARCH_SHOTS = int(os.getenv('WEIGHT_SEARCH_SHOTS', 0))
+WEIGHT_LIST_SHOTS = int(os.getenv('WEIGHT_LIST_SHOTS') or '0')
+WEIGHT_SEARCH_SHOTS = int(os.getenv('WEIGHT_SEARCH_SHOTS') or '0')
```

Comment:
This should better handle the cases where stuff flowing through Docker with null default values which then ends up setting the environment variables getting set to an empty string (giving us stuff like `WEIGHT_SEARCH_SHOTS=` (on the CLI) which then converts to `WEIGHT_SEARCH_SHOTS == ''` (in Python), which _then_ blows up `int()` since it doesn't know how to convert an empty string to an integer.
Oddly, it was looking like the `WEIGHT_LIST_SHOTS` style environment variables were getting set to zero-length empty strings, so the initial idea that the value would default to '0' wasn't true.

By using `or '0'` instead, we _hopefully_ bypass all those issues and make everything A-OK.